python/shotgun_model/data_handler_nav.py

```diff
@@ -30,7 +31,7 @@ class ShotgunNavDataHandler(ShotgunDataHandler):
     _SG_PATH_FIELD = "path"
     _SG_PARENT_PATH_FIELD = "parent_path"
 
-    def __init__(self, root_path, seed_entity_field, entity_fields, cache_path, parent):
+    def __init__(self, root_path, seed_entity_field, entity_fields, cache_path):
```

Comment:
Why was parent removed? You'll run the risk of having Qt delete the model at a later date and cause a crash or  memory leak in certain DCCs, as evidenced by the work I had to put it for releasing wf2.